src/moodleapi/Moodleapi.ts

```diff
@@ -0,0 +1,39 @@
+interface Response {
+  uuid: string
+  jwt: string
+}
+export class Moodleapi {
+  endpoint = 'local_raise_get_user'
+
+  async getUser(): Promise<Response> {
+    const userRequest = {
+      index: 0,
+      methodname: this.endpoint,
+      args: {
+      }
+    }
+    const url = this.getMoodlePluginMethodUrl(this.endpoint)
+    if (url === '') {
+      console.log('Invalid environment variables')
+    }
+    const response = await fetch(url, { method: 'POST', body: JSON.stringify([userRequest]) })
+    const responseJSON = await response.json()
+
```

Comment:
It would be good to do some error handling here. Unfortunately as we found with the service endpoint if there's an error Moodle will not send back an HTTP error code (e.g. `response.ok` will be `True`). Something like the following could be good to include so we don't get a parsing error later.

```javascript
    if (responseJSON.error !== undefined) {
      throw new Error(responseJSON.error)
    }
```

Reply:
Oh, you can assume it's there. On an error Moodle will include the field (you can test this real quick by changing the method string to something invalid).